GeoLib/Raster.cpp

```diff
@@ -148,9 +148,7 @@ double Raster::interpolateValueAtPoint(MathLib::Point3d const& pnt) const
 	for (unsigned j=0; j<4; ++j)
 	{
 		// check if neighbour pixel is still on the raster, otherwise substitute a no data value
-		if ( (xIdx + x_nb[j]) < 0 ||
-		     (yIdx + y_nb[j]) < 0 ||
```

Comment:
`xIdx` is a `std::size_t`, therefore the result of addition with an `int` is a `std::size_t`, which is never smaller than zero.